cli.js

```diff
@@ -90,6 +92,31 @@ const save = () => {
 
     // TODO handle creating files
     // TODO handle deleting files
+    const oldKeys = Object.keys(previousDocument);
+    const newKeys = Object.keys(currentDocument);
+    const removedKeys = oldKeys.filter((key) => !newKeys.includes(key));
+    removedKeys.forEach((key) => {
+      fs.unlinkSync(fullPath + '/' + previousDocument[key].name, (err) => {
+        process.on(err, () => { });
+      });
+    });
+
+    const addedKeys = newKeys.filter((key) => !oldKeys.includes(key));
+    console.log(addedKeys);
+    addedKeys.forEach((key) => {
+      let { name, text } = currentDocument[key];
+      if (!text) {
+        text = "";
+      }
+      fs.writeFileSync(name, text, (err) => {
+        if (err) {
+          if (err.code === 'ENOENT') {
```

Comment:
Does this happen? I would suggest we handle errors consistently, maybe just by throwing them for now.